change/@ni-nimble-components-083cf44c-68b4-43f6-b689-efd81c8228b4.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Haven't reviewed the rest yet but in regards to the following PR comment:

> buttonFillActivePrimaryColor and buttonFillAccentActiveColor (good riddance to inconsistent token names).

Those are inline with :token naming guidance:
https://github.com/ni/nimble/blob/8bb2c33f05e8e9a3db8a86804eabadbd7bc5d033/packages/nimble-components/CONTRIBUTING.md#L489-L498

element: button
part: fill
state: active, primary vs accent, active (the states are [sorted alphabetically](https://github.com/ni/nimble/blob/8bb2c33f05e8e9a3db8a86804eabadbd7bc5d033/packages/nimble-components/CONTRIBUTING.md#L497))
token_type: color

Reply:
It's good that it wasn't accidental, but it still looks and feels inconsistent. If we had used "AppearanceVariantAccent" and "AppearanceVariantPrimary" instead of just "Accent" and "Primary", that would have solved the inconsistency at the cost of having reasonable name lengths. A better solution might be to give "Active", "Hover", "Selected", "Disabled" their own category separate from all other state descriptors. The other states category could maybe be renamed to **variant**. Now that I've removed those two tokens, it looks like the rest are consistent with the modified scheme I described, so we could change our guidance if we want.

The only tokens I'm not quite clear on are those like `titlePlus1DisabledFontColor`. Apparently "Plus1" is not a state, but rather part of "titlePlus1" (which is the **part**). If so, then it's still consistent with my suggested change. If not, then it was already nonconforming.

Reply:
title-plus1 is the element / part correct.

I don't think I follow your proposal for the naming scheme. May help to give a concrete proposal. Feel free to make a separate PR if you feel passionate about it.

Reply:
I think it's a worthwhile change to make, so I created #1947